x-pack/plugins/watcher/client_integration/helpers/constants.ts

```diff
@@ -4,7 +4,7 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
-import { getWatch } from '../../../test/fixtures';
+import { getWatch } from '../../__fixtures__';
 
 export const WATCH_ID = 'my-test-watch';
 
```

Comment:
This constants.ts file was previously located within a `__jests__` directory. We might want to rename this file to something to identify it as used for tests, like `jest_constants.ts`

Reply:
Alright, done!